app/helpers/ember_rails_helper.rb

```diff
@@ -8,6 +8,11 @@ def render_ember_app(name, &block)
 
     head, body = markup_capturer.capture
 
-    render text: EmberCli[name].index_html(head: head, body: body).html_safe
+    template = EmberCli[name].index_html(head: head, body: body).html_safe
+    begin
+      render text: template
+    rescue ArgumentError
+      render body: template
```

Comment:
Sure :html would prob. be better - :body would be the closest equivalent to what :text does/did and I was not sure if the :html option was unwanted in some cases ... For compatibility I chose to have this rescue block - knowing that it's kind of ugly :/ 
4.0 seems not to support :body and :html options for render. http://guides.rubyonrails.org/v4.0/layouts_and_rendering.html#using-render